magic_moment_man.py

```diff
@@ -1,66 +1,45 @@
 import numpy as np
 import pandas as pd
-import engine
-import vehicle_params
+import itertools
 import multiprocessing
-import time
+from tqdm import tqdm
+from time import perf_counter
 
-solver = engine.Solver(vehicle_params.EasyDriver())
+import sys
+sys.path.append("..")
+import magic_moment_method.vehicle_params as vehicle_params
+import magic_moment_method.state_solver as state_solver
 
 def main():
+
     ### ~~~ SWEEP PARAMETERS FOR MMM ~~~ ###
     # NOTE: any parameter in the vehicle_params file can be swept as well
     # NOTE: guesstimation based from TTC on maximum tire saturation slip angle
     peak_slip_angle = 18 * np.pi / 180 # rad
-    refinement = 21
+    refinement = 3
 
     s_dot_sweep = [15] # velocity sweep in path tangential direction (total velocity)
-    body_slip_sweep = np.linspace(-peak_slip_angle, peak_slip_angle, refinement)
+    body_slip_sweep = np.linspace(-10 * np.pi / 180, 10 * np.pi / 180, refinement)
```

Comment:
Reason we're cutting at 10 now?

Reply:
See negative wheel slip below that I said was a long story - saturation hits at body slip of 10 with that fix